aiida-registry-app/src/App.jsx

```diff
@@ -0,0 +1,268 @@
+              />
+          )}
+
+          </p>
+
+          <p>{value.metadata.description}</p>
+          <ul className="plugin-info">
+            <li>
+          <a href={value.code_home}>Source Code</a>
+            </li>
+          {value.documentation_url && (
+            <li>
+            <a href={value.documentation_url}>Documentation</a>
+            </li>
+          )}
+          <li>
+          <a href={`/${key}`}>Plugin details</a>
+          </li>
+
+          </ul>
+
+          {value.summaryinfo && (
+            <>
+              <p className="summaryinfo">
+              {value.summaryinfo.map((summaryinfoelem) => (
+                <span className="badge" key={summaryinfoelem.text}>
+                  <span className={`badge-left ${summaryinfoelem.colorclass}`}>
+                    {summaryinfoelem.text}
+                  </span>
+                  <span className="badge-right">{summaryinfoelem.count}</span>
+                </span>
+              ))}
+              </p>
+            </>
+          )}
+
+        </div>
+      ))}
+    </div>
+    </main>
+  );
+}
+
+function Details() {
```

Comment:
I don't have much experience with javascript, function name starting with capitalized letters is not good I assume?

Reply:
This is true, javascript naming convention is camel case, but since this function is a react component, it is common to name it with capital letters (Pascal case).